README.md

```diff
@@ -0,0 +1,78 @@
+# Apache Beam starter for Python
+
+If you want to clone this repository to start your own project,
+you can choose the license you prefer and feel free to delete anything related to the license you are dropping.
+
+## Before you begin
+
+Make sure you have a [Python 3](https://www.python.org/) development environment ready.
+If you don't, you can follow the instructions in the
+[Python 3 Installation & Setup Guide](https://realpython.com/installing-python/).
```

Comment:
You might consider linking to the official Python docs, instead of RealPython. Or at least use link text that indicates it's RealPython.

Reply:
Changed it to the Python downloads page